src/scenes/home/benefit/benefit.js

```diff
@@ -94,7 +111,7 @@ const Benefit = () => (
         Business Casual
         </p>
         <br />
-        <h6>Event Chairs:</h6>
+        <h6>Event Co-Chairs:</h6>
         <p>
         Chris Dawson & David Molina
         </p>
```

Comment:
It may be helpful to apply a class to this element (the `<p>` that is wrapping all of these logos) and give it `display: flex`.  That way you can use the flex properties such as `flex-flow: wrap`, `justify-content`, and `align-items` to center things and control the wrapping.

It probably should not be a `<p>` element, either, as that is not semantically correct - this is not a paragraph of text.